### ISSUE TITLE
Allow cyberwatch-cli to use information from api.conf configuration file

### ISSUE THREAD

**Issue by jljuan** [Cyberwatch/cyberwatch_api_toolbox#258]
It should be interesting that the CLI take into account the API configuration file if present in order to avoid to use the inline options visibles into command line history.

**Comment by hezanathos** [comment]
Yes, it should be nice to have use this file as all other API usage are compatible.

Currently env variables can be used to avoid inline options but this is still something we will look at.

**Comment by hezanathos** [comment]
To be done afer https://github.com/Cyberwatch/cyberwatch_api_toolbox/issues/219
